.snippets/code/develop/toolkit/api-libraries/polkadot-js-sdk/read-state.js

```diff
@@ -0,0 +1,14 @@
+// Example address
+const ADDRESS = '5DTestUPts3kjeXSTMyerHihn1uwMfLj8vU8sqF7qYrFabHE';
+
+// Get current timestamp
+const timestamp = await api.query.timestamp.now();
+
+// Get account information
+const { nonce, data: balance } = await api.query.system.account(ADDRESS);
```

Comment:
```suggestion
const address = '5DTestUPts3kjeXSTMyerHihn1uwMfLj8vU8sqF7qYrFabHE';

// Get current timestamp
const timestamp = await api.query.timestamp.now();

// Get account information
const { nonce, data: balance } = await api.query.system.account(address);
```

No need to use all uppercase for variables unless they are exported constants

Reply:
Hey sorry for jumping in here, but I believe it's okay to have those variables in uppercase. AFAIK, in JS, it's a common convention to use uppercase + `_` between words when we are handling variables that are intended to remain constant across the snippet

Maybe I'm mistaken, but this is what I get to understand from this answer - https://stackoverflow.com/a/51884773